boards/x86/intel_rpl/intel_rpl_s_crb_defconfig

```diff
@@ -13,3 +13,6 @@ CONFIG_X2APIC=y
 CONFIG_SMP=y
 CONFIG_BUILD_OUTPUT_EFI=y
 CONFIG_BUILD_NO_GAP_FILL=y
+CONFIG_HEAP_MEM_POOL_SIZE=64000000
+CONFIG_MAIN_STACK_SIZE=320000
```

Comment:
It's quite odd to see the main stack explicitly set by the board - the requirements should pretty much 100% depend on the application based on what it does through its `main()` function. Why does it need to be set here instead of in the application?

Reply:
Removed

Reply:
Don't we have set it already?

https://github.com/zephyrproject-rtos/zephyr/blob/f69641f7d204864aa26f8bdd9fecab259e535da2/boards/x86/intel_rpl/Kconfig.defconfig#L33-L37

Reply:
@finikorg yes, I don't think any board should be doing that. The requirements for the `main()` function should be only set by the app. `Kconfig.defconfig` is at least a little bit better than `<board>_defconfig` since it's just modifying the default value, i.e. the app can still override it.